scripts/tests/twister/test_runner.py

```diff
@@ -1958,7 +1958,7 @@ def mock_open(fname, *args, **kwargs):
         ['INFO      20/25 dummy platform' \
          '            dummy.testsuite.name' \
          '                               PASSED' \
-         ' (dummy handler type: dummy dut, 60.000s)'],
+         ' (dummy handler type: dummy dut, 1m 0.000s)'],
```

Comment:
could you please test more of the new combinations, e.g.:
* seconds <60,
* non-zero minutes and seconds,
* hours + 0 minutes + some seconds.

Reply:
Done, i added 3 new unit tests